paderbox/transform/module_stft.py

```diff
@@ -344,6 +347,8 @@ def sample_index_to_stft_frame_index(sample, window_length, shift, fading='full'
 
     >>> [sample_index_to_stft_frame_index(i, 8, 1, fading=None) for i in range(12)]
     [0, 0, 0, 0, 1, 2, 3, 4, 5, 6, 7, 8]
+    >>> sample_index_to_stft_frame_index(np.arange(12), 8, 1, fading=None)
```

Comment:
Add a dtype for arange. The Windows test fails, because the default is int32 on windows.